index.html

```diff
@@ -283,6 +283,47 @@ <h2 class="headline-text">About Myself</h2>
             </ul>
           </div>
         </section>
+
+        <section id="contact-form">
+          <div class="contact-wrapper">
+            <div>
+              <h2 class="contact-headline">Contact me</h2>
+              <p class="contact-text">
+                If you have an application you are interested in developing, a
+                feature that you need built or a project that needs coding. I'd
+                love to help with it
+              </p>
+            </div>
+            <form
+              action="https://formspree.io/f/mjvzkyre"
+              method="post"
+              id="form"
+            >
+              <input
+                type="text"
+                id="text"
+                maxlength="30"
+                placeholder="Enter your name"
+                required
+              />
+              <input
+                type="email"
+                name="email"
+                id="email"
+                placeholder="Enter your email"
+                required
+              />
+              <textarea
+                name="comment"
+                id="comment"
+                maxlength="500"
+                placeholder="Write your message here"
+                required
+              ></textarea>
+              <button type="submit" id="form-btn">Get In Touch</button>
+            </form>
+          </div>
+        </section>
```

Comment:
- [ ] Good job with your project. I noticed that you miss the **Horizontal line** at the bottom to stick as much as possible with the Figma template design. MAke sure you add it.
![Screenshot from 2022-08-24 18-19-39](https://user-images.githubusercontent.com/83042742/186490878-3a2540c6-c4d1-441e-a723-c37030a557e9.png)